draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+    - If the type auxiliary input is set :math:`\mathsf{type}_\mathbb{P}` = :math:`\mathcal{V}^\mathsf{Orchard}`
+        - NoteCommitment has a “compatibility” path that computes the note commitment as in plain Orchard (i.e.: without including the type)
+        - This path also uses the original domain separator for ZEC note commitment
+    - Else, 
+        - The NoteCommitment adds the type, :math:`\mathsf{type}_\mathbb{P}`, as a final “chunk” of the Sinsemilla commitment
+        - The NoteCommitment uses a different domain separator for ZSA note commitment
+
+
+Backward Compatibility
+----------------------
+
+In order to have a "clean" backwards compatibility with the ZEC notes, we have designed the circuit to support both ZEC and ZSA notes. As we specify above, there are three main reasons we can do this:
+- The input notes with a type denote the ZSA custom assets, generating a note commitment that includes the type; whereas the notes without a type, denote the ZEC notes, and generate a note commitment that does not include the type, in order to maintain the referencability to the Merkle tree
+- The value commitment is abstracted to allow for the value base-point as a variable private input to the proof
+- The ZEC-based actions will still include dummy input notes, whereas the ZSA-based actions will include split input notes
+
+Burn Mechanism
+==============
+The burn mechanism may be needed for off-boarding the wrapped assets from the chain, or enabling advanced tokenomics on native tokens. It is part of the Issuance/Burn protocol, but given that it can be seen as an extension of the Transfer protocol, we add it here for readability.
+
+In essence, the burn mechanism is a transparent / revealing extension to the transfer protocol that enables a specific amount of any asset type to be sent into “oblivion”. Our burn mechanism does NOT send assets to a non-spendable address, it simply reduces the total number of assets in circulation at the consensus level. It is enforced at the consensus level, by using an extension of the value balance mechanism used for ZEC assets.
+
+First, contrary to the strict transfer transaction, we allow the sender to include a :math:`\mathsf{valueBalvalueBalance_{type}}` variable for every asset type that is being burnt. As we will show in the transaction structure, this is separate from the regular :math:`\mathsf{valueBalance^Orchard}` that is the default transparent value for the ZEC asset.
+
+For every custom asset that is burnt, we add to the `assetBurn` vector the tuple :math:`(\mathsf{valueBalance_{type}, type}_\mathbb{P})` such that the validator of the transaction can compute the value commitment with the corresponding value base point of that asset. This ensures that the values are all balanced out with respect to the asset types in the transfer.
+
+
+:math:`\mathsf{assetBurn = [(v^{type}, type_\mathbb{P})}| \forall \mathsf{type}_\mathbb{P}  \textit{ s.t.}\mathsf{v^{type}\neq 0}]`
```

Comment:
Is there a consensus rule that each $v^{type} \neq 0$?

Reply:
This has been added in now, thanks!